Doc/library/pathlib.rst

```diff
@@ -1217,6 +1217,8 @@ call fails (for example because the path doesn't exist).
       >>> p.absolute()
       PosixPath('/home/antoine/pathlib/tests')
 
+   .. versionadded:: 3.11
```

Comment:
```suggestion
   .. versionadded:: 3.4
```
But per Brandt, this likely isn't needed at all.

A

Reply:
yeah my bad i was committing changes when he posted and I didn;t see that

Reply:
This is marked as resolved, though you haven't pushed the change yet -- is this intentional?

A

Reply:
isn't it version 3.11?

Reply:
No i didn't understand the term!
i just clicked on it and was waiting for your reply then I saw it was changed to unresolved;😅